pkg/volume/gce_pd/attacher.go

```diff
@@ -165,6 +165,9 @@ func (attacher *gcePersistentDiskAttacher) MountDevice(spec *volume.Spec, device
 	options := []string{}
 	if readOnly {
 		options = append(options, "ro")
+	} else {
+		// as per https://cloud.google.com/compute/docs/disks/add-persistent-disk#formatting
```

Comment:
You'd have to explain more what you're thinking.

On Thu, Jul 7, 2016 at 12:14 AM, Michael Rubin <notifications@github.com>
wrote:

> In pkg/volume/gce_pd/attacher.go
> <https://github.com/kubernetes/kubernetes/pull/28448#discussion_r69860194>
> :
>
> > @@ -165,6 +165,9 @@ func (attacher *gcePersistentDiskAttacher) MountDevice(spec *volume.Spec, device
> >  	options := []string{}
> >  	if readOnly {
> >  		options = append(options, "ro")
> > +	} else {
> > +		// as per https://cloud.google.com/compute/docs/disks/add-persistent-disk#formatting
>
> @thockin <https://github.com/thockin> @saad-ali
> <https://github.com/saad-ali> @ciwang <https://github.com/ciwang> Can't
> you just use "file -s"?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28448/files/8efefab9a3fdc4906ca53d4a23d22c3846d0610b#r69860194>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFVgVMuhjJb2CDIZeLUmPmD6RanfX6Dvks5qTKdggaJpZM4JECrD>
> .
>